function.js

```diff
@@ -0,0 +1,145 @@
+// // Перепишите функцию, чтобы она делала то же самое, но без if, в одну строку.
+// // Сделайте два варианта функции checkAge:
+// // Используя оператор ?
+// // Используя оператор ||
+
+function checkAge(age) {
+    return (age >= 18) ? true : confirm("Родители разрешили?")
+};
+
+function checkAge(age) {
+    return (age >= 18) || confirm("Родители разрешили?")
+};
+
+function checkAge(age) ? () => true: () confirm("Родители разрешили?");
+
+
+// // Напишите функцию min(a,b), которая возвращает меньшее из чисел a и b.
+function getMin(a, b) {
+    return (a < b) ? a : b
+};
+
+// Напишите функцию, которая принимает 2 числа и возвращает 1, если первое число больше, чем второе; -1, если первое число меньше, чем второе, и 0, если числа равны.
+
+function getVar(a, b) {
+    if (a > b) {
+        return 1;
+    } else(b > a) {
+        return -1;
+    };
+}
+console.log(getVar(2, 3));
+
+let getVar = (a, b) => {
+    return (a > b) ? 1 : (b > a) ? -1
+};
+
+// Напишите функцию, которая принимает 2 параметра: длину и ширину прямоугольника и 
+// вычисляет его площадь.Если в функцию передали 1 параметр, то функция вычисляет площадь квадрата.
+
+function getSquare(length, width = l) {
+    if (length >= 0 && width >= 0) ?
+        return length * width: "Mistake";
+};
+console.log(getSquare(5, 2));
+console.log(getSquare(10));
+console.log(getSquare(10, "w"));
+
+// Написать функцию, получающую на вход два числа. Если оба числа чётные - функция возвращает их произведение. 
+// Если оба числа нечётные - функция возвращает их сумму.
+// Если одно из чисел чётное, а второе нечётное - функция возвращает это нечётное число.
+
+function getResult(a, b) {
+    if (!(a % 2) && !(b % 2)) {
+        return a * b;
+    } else if ((a % 2) && (b % 2)) {
+        return a + b;
+    } else if (!(a % 2) || (b % 2)) {
+        return b;
+    } else if (!(b % 2) || (a % 2)) {
+        return a;
+    }
+}
+
+console.log(getResult(5, 8));
+console.log(getResult(7, 9));
+console.log(getResult(8, 10));
+
+// Написать функцию, которая принимает 1 аргумент (возраст пользователя)
+// и возвращает boolean значение  - можно ему на сайт или нет.( если 18 и больше лет)
+
+function getAge(age) {
+    if (age >= 18) {
+        return true
+    } else {
+        return "Not access"
+    }
```

Comment:
return true or false

rewrite it in 1 line

Reply:
edited